server/src/test/java/org/elasticsearch/index/mapper/DocumentParserTests.java

```diff
@@ -607,10 +607,8 @@ MapperService createMapperService() throws Exception {
     }
 
     // creates an object mapper, which is about 100x harder than it should be....
-    ObjectMapper createObjectMapper(MapperService mapperService, String name) {
-        DocumentMapper docMapper = mapperService.documentMapper();
-        ParseContext context = new ParseContext.InternalParseContext(docMapper.mappers(), mapperService.getIndexSettings(), null,
-            null, null, null);
+    private static ObjectMapper createObjectMapper(String name) {
+        ParseContext context = new TestParseContext();
```

Comment:
Do you even need a ParseContext here? Could just use a ContentPath and pass it directly to build()? In which case the comment at the top of the method is not really true anymore!

Reply:
no we don't I had already made this change but I lost it through different attempts. Thanks for bringing this up